tsl/test/sql/data/partialize_finalize_data.csv

```diff
@@ -0,0 +1,14 @@
+12,,\x000000000000000000000000000000000000000000000000000000000000000000000000,,,\x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000,\x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
+1,40,\x0000000000000001000100000000000000280000000000000000000000010000000000000000,\x6162636465,,\x000000000000000100010000000000000028000100000000000006400000000000000000000000010000000000000000,\x000000000000000100010000000000000028000100000000000006400000000000000000000000010000000000000000
+1,50,\x0000000000000001000100000000000000320000000000000000000000010000000000000000,,\x00011f1c121ac400,\x000000000000000100010000000000000032000100000000000009c40000000000000000000000010000000000000000,\x000000000000000100010000000000000032000100000000000009c40000000000000000000000010000000000000000
+11,,\x000000000000000000000000000000000000000000000000000000000000000000000000,\x68656c6c6f,\x00011f2dac373800,\x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000,\x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
+2,20,\x0000000000000001000100000000000000140000000000000000000000010000000000000000,\x68656c6c6f,\x00011f1c121ac400,\x000000000000000100010000000000000014000100000000000001900000000000000000000000010000000000000000,\x000000000000000100010000000000000014000100000000000001900000000000000000000000010000000000000000
+3,40,\x0000000000000001000100000000000000280000000000000000000000010000000000000000,,,\x000000000000000100010000000000000028000100000000000006400000000000000000000000010000000000000000,\x000000000000000100010000000000000028000100000000000006400000000000000000000000010000000000000000
+3,50,\x0000000000000001000100000000000000320000000000000000000000010000000000000000,,,\x000000000000000100010000000000000032000100000000000009c40000000000000000000000010000000000000000,\x000000000000000100010000000000000032000100000000000009c40000000000000000000000010000000000000000
+2,30,\x00000000000000010001000000000000001e0000000000000000000000010000000000000000,\x68656c6c6f,\x00011f1c121ac400,\x00000000000000010001000000000000001e000100000000000003840000000000000000000000010000000000000000,\x00000000000000010001000000000000001e000100000000000003840000000000000000000000010000000000000000
+1,20,\x0000000000000001000100000000000000140000000000000000000000010000000000000000,\x616263,\x00011f302ff22400,\x000000000000000100010000000000000014000100000000000001900000000000000000000000010000000000000000,\x000000000000000100010000000000000014000100000000000001900000000000000000000000010000000000000000
+12,10,\x00000000000000010001000000000000000a0000000000000000000000010000000000000000,\x68656c6c6f,\x00011f2dac373800,\x00000000000000010001000000000000000a000100000000000000640000000000000000000000010000000000000000,\x00000000000000010001000000000000000a000100000000000000640000000000000000000000010000000000000000
```

Comment:
How did you create this file? Might be good to add that script separately, with instructions on how to use it to generate partials for testing.

Reply:
It was generated by the same `partialize_finalize.sql` file it is read from. When executing `TRUNCATE TABLE t1;` it overwrites the table with the same data generated from a PG13 instance.

Reply:
Well, if a developer need to extend this test case, or fix something that was broken, it would be good to have a script and/or instructions to look at to figure out how to do the necessary changes.